index.html

```diff
@@ -1261,66 +1261,68 @@ <h2>Characters</h2>
-	</div>
-
-    <p>The relationship between code points and graphemes can be complex. In most cases, a code point sequence that forms a single grapheme should be treated as a single textual unit. For example, when cursoring across text, an entire grapheme should select together. It shouldn't be possible to cursor into the "middle" of a grapheme or delete only a part of user-perceived character. Because the relationship is not one-to-one between code points and graphemes and because the relationship can be somewhat complex, [[Unicode]] defines a specific type of grapheme: the <a>extended grapheme cluster</a> which most closely matches the mapping of the underlying logical character sequence to a user-perceived character. When referring to 'graphemes' in this document, we mean extended grapheme clusters (unless otherwise called out).</p>
-
-    <aside class=example title="Example of the difference between graphemes and code points">
-		<p>Returning to the example above, the Hindi word for Unicode is made of four graphemes:</p>
-		<p class=bigtext lang=hi>&#x092F;&#x0942;&nbsp;<span style="color:red">&#x0928;&#x093F;</span>&nbsp;&#x0915;&#x094B;&nbsp;&#x0921;</p>
-		<p>Several of these graphemes are made up of more than one Unicode character because of the way that the Devanagari script works. In Devanagari, the basic set of "letters" are syllables ending with the short 'a' vowel sound. When you want to use a different vowel, you add a combining vowel character that changes the shape of the grapheme. The red text in the example above is the syllable "ni" in "Unicode". It is made of two characters: U+0928 (the syllable "na") and U+093F (combining "short i" sound):</p>
+    <p>The relationship between [=code points=] and [=visual text units=] can be complex. In most cases, a sequence of [=code points=] that forms a single [=visual text unit=] should be processed as a single, indivisible textual unit. For example, when cursoring across text, the cursor should "jump across" or select the entire visual text unit (and its underlying code points) together. It shouldn't be possible to cursor into the "middle" of a visual text unit.</p>
+    
+	<p>[[Unicode]] provides a definition for approximately computing the boundaries in a sequence of characters between visual text units. The boundaries defined by [[Unicode]] are called [=grapheme clusters=]. Unless otherwise specified, the term [=grapheme cluster=] in this document refers to what [[UAX29]] refers to as an "extended default grapheme cluster".</p>
+	
+    <aside class="example" title="Example of the difference between grapheme clusters and code points">
+		<p>Returning to [[[#visual-text-unit-example]]] above, the Hindi word for "Unicode" is made of four visual text units:</p>
+		<p class="bigtext" lang="hi">&#x092F;&#x0942;&nbsp;<span style="color:red"><strong>&#x0928;&#x093F;</strong></span>&nbsp;&#x0915;&#x094B;&nbsp;&#x0921;</p>
```

Comment:
I suspect we'll have an accessibility issue here. How will a user with a screen reader see which characters you are highlighting?  I wonder whether using the `<mark>` element would work better?  We can style that however we want (although a background colour change may be easier on the eye than the multi-colour and bolding effects?).

Reply:
> U+0928 (the syllable "na") and U+093F 

Why not use `<span class="codepoint" translate="no"><bdi lang="ml">&#x0928;</bdi> <span class="uname">U+0928 DEVANAGARI LETTER NA</span></span> (the syllable "na") and <span class="codepoint" translate="no"><bdi lang="ml">&#x093F;</bdi> <span class="uname">U+093F DEVANAGARI VOWEL SIGN I​</span></span>`